scala.html.markdown

```diff
@@ -253,16 +253,20 @@ weirdSum(2, 4) // => 16
 // def that surrounds it.
 // WARNING: Using return in Scala is error-prone and should be avoided.
 // It has no effect on anonymous functions. For example:
-def foo(x: Int): Int = {
-  val anonFunc: Int => Int = { z =>
+def addTenButMaybeTwelve(x: Int): Int = {
+  val anonMaybeAddTwo: Int => Int = { z =>
     if (z > 5)
-      return z // This line makes z the return value of foo!
+      return z // This line makes z the return value of addTenButMaybeTwelve!
     else
       z + 2    // This line is the return value of anonFunc
   }
-  anonFunc(x)  // This line is the return value of foo
+  anonMaybeAddTwo(x) + 10  // This line is the return value of foo
```

Comment:
yeah but it's not actually the return value - that'd mean the +10 gets applied everytime which isn't true.

we should maybe add "return" for the else as well